packages/esbuild-plugin/.babelrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/env", "@babel/typescript"]
+}
```

Comment:
Totally optional: 
I think the values here are the same for all projects, right? Can we extract this file into one package?